en/peripherals/dshot.md

```diff
@@ -46,16 +47,14 @@ While DShot is enabled on either or both groups, normal PWM is supported on any
 Remove propellers before changing ESC configuration parameters!
 :::
 
-Enable DShot with the [DSHOT_CONFIG](../advanced_config/parameter_reference.md#DSHOT_CONFIG) parameter (if the parameter does not exist, the board does not support DShot).
+Enable DShot for your required outputs in the [Actuator Configuration](../config/actuators.md).
 
 DShot comes with different speed options: *DShot150*, *DShot300*, *DShot600* and *DShot1200*, where the number indicates the speed in kilo-bits/second.
-You should set the parameter to the highest speed supported by your ESC (according to its datasheet) and then reboot the vehicle.
+You should set the parameter to the highest speed supported by your ESC (according to its datasheet).
 
 Then connect the battery and arm the vehicle.
 The ESCs should initialize and the motors turn in the correct directions.
-- If the motors do not spin in the correct direction (for the [selected airframe](../airframes/airframe_reference.md)), reverse them by sending an [ESC Command](#commands).
-- Adjust [DSHOT_MIN](../advanced_config/parameter_reference.md#DSHOT_MIN) so that the motors spin at lowest throttle (but the vehicle does not take off).
-
+- If the motors do not spin in the correct direction (for the [selected airframe](../airframes/airframe_reference.md)) you can reverse them in the UI using the **Set Spin Direction** options (or reverse them by sending an [ESC Command](#commands)).
```

Comment:
I couldn't see "Set Spin Direction" on my UI - is this an advanced setting option?

Reply:
It will show up if you select dshot and assigned motors.